### ISSUE TITLE
Windows installer 6.0.1: Directory is not empty.

### ISSUE THREAD

**Issue by LINAK-thlu** [spyder-ide/spyder#22569]
### Issue Report Checklist

* [ x] Searched the [issues page](https://github.com/spyder-ide/spyder/issues?q=is%3Aissue) for similar reports
* [ x] Completed the **Problem Description**, **Steps to Reproduce** and **Version** sections below

## Problem Description
When installing version 6.0.1 on top of 6.0.0 using the Windows installer this error _Directory is not empty_ shows:

![image](https://github.com/user-attachments/assets/c226648d-12e9-439a-829f-33e7e7406cb9)

### What steps reproduce the problem?

1.  Install 6.0.0
2.  Install 6.0.1

**Comment by ccordoba12** [comment]
Hey @LINAK-thlu, thanks for reporting. I'm not so sure if there's something we could do about this. For now, you need to manually remove Spyder 6.0.0 before installing 6.0.1. However, Spyder will offer you to automatically update to 6.0.1 a few minutes after you start it, so you wouldn't need to do that.

@mrclary, what do you think about this one?

**Comment by mrclary** [comment]
> Hey @LINAK-thlu, thanks for reporting. I'm not so sure if there's something we could do about this. For now, you need to manually remove Spyder 6.0.0 before installing 6.0.1. However, Spyder will offer you to automatically update to 6.0.1 a few minutes after you start it, so you wouldn't need to do that.

That is correct. Our installer will not install over an existing Spyder installation. Your options are (in preferential order):
1. If you have used our installer already to install Spyder 6.0.0, then Spyder may check for updates on startup or you can manually check for updates from the menubar: Help -> Check for updates... which will lead to Spyder doing an in-place update.
2. You can uninstall Spyder 6.0.0 using the Windows program uninstaller, delete the `spyder-6` directory if it is not fully removed, and install Spyder 6.0.1.

**Comment by ccordoba12** [comment]
> Our installer will not install over an existing Spyder installation.

But is it possible to switch something in Constructor to do that?

**Comment by mrclary** [comment]
> > Our installer will not install over an existing Spyder installation.
> 
> But is it possible to switch something in Constructor to do that?

Not at this time. https://github.com/conda/constructor/issues/679
